frontend/device/generic/device.lua

```diff
@@ -238,6 +238,10 @@ function Device:setScreenDPI(dpi_override)
     self.input.gesture_detector:init()
 end
 
+function Device:getDisplayDPI()
```

Comment:
May be it should be named `getScreenRealDPI()` (or something else instead of "real", @Frenzie ?)

Reply:
getDeviceScreenDPI? Assuming it doesn't relate to the setScreenDPI function right above.